<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="vaibhavagarwal" name="in.redbus.Tests.BusHirePageTest" tests="8" failures="1" timestamp="05 Jul 2021 09:18:20 GMT" time="179.115" errors="0">
  <testcase name="whenUserClickedOnLocalMenu" time="18.873" classname="in.redbus.Tests.BusHirePageTest"/>
  <testcase name="whenUserEntersPassengerNumberGreaterThan60ForOutstation" time="35.848" classname="in.redbus.Tests.BusHirePageTest"/>
  <testcase name="whenUserDoesNotEnterTillDateAndTimeForOutstation" time="7.608" classname="in.redbus.Tests.BusHirePageTest">
    <skipped/>
  </testcase> <!-- whenUserDoesNotEnterTillDateAndTimeForOutstation -->
  <testcase name="whenUserDoesNotEnterToLocationForOutstation" time="30.895" classname="in.redbus.Tests.BusHirePageTest"/>
  <testcase name="whenUserDoesNotEnterFromDateAndTimeForOutstation" time="7.832" classname="in.redbus.Tests.BusHirePageTest">
    <skipped/>
  </testcase> <!-- whenUserDoesNotEnterFromDateAndTimeForOutstation -->
  <testcase name="whenUserDoesNotEnterFromLocationForOutstation" time="32.750" classname="in.redbus.Tests.BusHirePageTest"/>
  <testcase name="whenUserEntersValidInformationForOutstation" time="32.508" classname="in.redbus.Tests.BusHirePageTest">
    <failure type="java.lang.AssertionError" message="Assertion on actual and expected result of header summary label expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Assertion on actual and expected result of header summary label expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at in.redbus.Tests.BusHirePageTest.whenUserEntersValidInformationForOutstation(BusHirePageTest.java:89)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- whenUserEntersValidInformationForOutstation -->
  <testcase name="whenUserClickedOnAirportMenu" time="12.801" classname="in.redbus.Tests.BusHirePageTest"/>
</testsuite> <!-- in.redbus.Tests.BusHirePageTest -->
